tests/postthrow.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Check if throwing exception in post hook after IO operation will finish gracefully
+--EXTENSIONS--
+otel_instrumentation
+--FILE--
+<?php
+\OpenTelemetry\Instrumentation\hook(
+    null,
+    'helloWorld',
+    fn() => var_dump('pre'),
+    fn(object|string|null $scope, array $params, mixed $returnValue, ?Throwable $throwable) => throw new Exception('error'));
+
+
+function helloWorld() {
+  scandir(".");
+}
+
+helloWorld();
+
+--EXPECT--
+string(3) "pre"
+
+Fatal error: Uncaught Exception: error in /home/vagrant/opentelemetry-php-instrumentation/tests/postthrow.php:6
```

Comment:
I think this will need to be made more generic with `EXPECTF`

Reply:
Yes, right